tests/test_database_manager.py

```diff
@@ -0,0 +1,93 @@
+"""
+test_database_manager.py - UT for the databaseManager
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+
+This module is built on top of the Pydle system.
+"""
+import pytest, pytest_asyncio, pyodbc
+from Modules.database_manager import DatabaseManager
+
+pytestmark = pytest.mark.asyncio
+
+
+# noinspection PyProtectedMember
+class TestStuff(object):
+    manager = DatabaseManager()
+
+    @classmethod
+    def setup_class(cls):

```

Comment:
We can make use of PyTest's fixtures to do class/function setup routines. using `setup_class` is the legacy method

Reply:
done
hopefully